config/initializers/permissions.rb

```diff
@@ -43,6 +43,19 @@ def edit_project_hash
 
 OpenProject::AccessControl.map do |map|
   map.project_module nil, order: 100 do
+    map.permission :add_project,
+                   { projects: %i[new create] },
+                   require: :loggedin,
+                   global: true
+
+    map.permission :add_user,
+                   {
+                     users: %i[index show new create edit update],
+                     admin: %i[index]
+                   },
+                   require: :loggedin,
+                   global: true
```

Comment:
It would probably help if we introduce a global_module method on the `map` object which can then take all the global permissions.